foodsaving/conversations/emails.py

```diff
@@ -22,17 +35,73 @@ def prepare_conversation_message_notification(user, message):
     reply_to = formataddr((reply_to_name, '{}@{}'.format(local_part, settings.SPARKPOST_RELAY_DOMAIN)))
     from_email = formataddr((message.author.display_name, settings.DEFAULT_FROM_EMAIL))
 
-    with translation.override(user.language):
-        return prepare_email(
-            'conversation_message_notification',
-            from_email=from_email,
-            user=user,
-            reply_to=[reply_to],
-            context={
-                'conversation_name': conversation_name,
-                'author': message.author,
-                'message_content': message.content_rendered(),
-                'conversation_url': group_wall_url(group),
-                'mute_url': conversation_mute_url(group, message.conversation)
+    return prepare_email(
```

Comment:
Why did you remove the translation override?

Reply:
It seems unnecessary, as `prepare_email` has its own translation override.

Reply:
Ah, yes I see. Makes sense. And for the other one it does translation stuff in the preparation so it does need it.